serde_derive/src/de.rs

```diff
@@ -795,9 +795,19 @@ fn deserialize_adjacently_tagged_enum(
 
     let expecting = format!("adjacently tagged enum {}", params.type_name());
     let type_name = cattrs.name().deserialize_name();
+    let deny_unknown_fields = cattrs.deny_unknown_fields();
+
+    /// If unknown fields are allowed, we pick the visitor that can
```

Comment:
In my IDE (VSCode), `///` comments on local vars are supposed to be showing up, like with JSDoc comments on private functions or members.